src/message.ts

```diff
@@ -452,10 +452,11 @@ export class Message implements Sayable {
     // atList here is trying to fill in mentionList in function multipulAt()
     atList
       .filter(str => str.includes('@'))
-      .map(str => multipleAt(str))
-      .filter(str => !!str) // filter blank string
+      .map(str => {
+        mentionList = mentionList.concat(multipleAt(str) || [])
```

Comment:
No. this is also a side effect to `mentionList` inside `map()` .

Please DO NOT modify `mentionList` inside any `map()` function.